src/sas/sasgui/perspectives/fitting/fitpage.py

```diff
@@ -1238,7 +1238,12 @@ def _on_select_model(self, event=None):
             custom_model = CUSTOM_MODEL
             mod_cat = self.categorybox.GetStringSelection()
             if mod_cat == custom_model:
+                ################This is where the model dies###############
```

Comment:
I'm guessing this was a placeholder comment to show where the issue was. Can it be removed?